src/components/misc/InfoList.jsx

```diff
@@ -39,6 +39,12 @@ export default class InfoList extends React.Component {
                                 }
                             }
                         }
+                        if(item.msgDatetimes) {
```

Comment:
I don't mind this at this point, since it's already implemented, but the same thing should have been possible without changing `InfoList` by adding a `msgId` and the date in `msgValues`, and in the localization of the string using `{ myDate, date }` (where the `date` is a keyword telling the localization framework to format it as a date).

Reply:
Interesting. I tried this but it didn't work. It did work in another place with the <FormattedMessage> React-component for number formatting though. Could it be that it doesn't work with inject-intl?

Reply:
It definitely should work, but lets not dive to deeply into it now. This is fine!